Waffle 1.7.1

Thinking-waffle opened the pull request.

Thinking-waffle committed:
USA starts with M1 Garand

Thinking-waffle committed:
Fixing some commanders

Thinking-waffle committed:
fixing small bugs related to generals

Thinking-waffle committed:
changed generic portrait to avoid clones

Thinking-waffle committed:
decision to recreate Belgium with Flanders and Wallonia

Thinking-waffle committed:
late nuclear techs boost nuke production speed.

Thinking-waffle committed:
improved belgium recreation

Thinking-waffle committed:
reformed belgium get reformed penalties

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-1.7.1

Thinking-waffle committed:
Soviet will rush service by requirement only when at war (and later)

Thinking-waffle committed:
Commonweath countries will use British ships

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-1.7.1

Thinking-waffle committed:
build cost of variants +5%->+3% + tanks

Thinking-waffle committed:
variants 3->+2%IC

Thinking-waffle committed:
Update czechoslovakia.txt

Thinking-waffle committed:
Generic decisions to coordonate manufacturers

Thinking-waffle committed:
fixing typo

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-1.7.1

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-1.7.1

Thinking-waffle committed:
REUPLOAD ALL THE THINGS

Thinking-waffle committed:
apparently definition.csv is not needed

Thinking-waffle committed:
Removing duplicate icon, fixing manufacturer traits

Thinking-waffle committed:
changes of 1.8 + plutozz color change scotland

Thinking-waffle committed:
advisors quebec canada

Thinking-waffle committed:
mainly belgian things

-Improved Belgian unification
-Fixed portrait of Jacquemotte
-Removed some duplicate generics from Flanders and Wallonia
-Added FT of TFV australia to fix a bug

SpicyAlfredo reviewed (commented):
Why delete the mod file in the pull?

Thinking-waffle committed:
improvement Belgian reformation

Thinking-waffle committed:
France should recognize that Flanders and Wallonia can be conquered.

Thinking-waffle committed:
France now nows that wallonia is a thing, and it can be annexed.

Thinking-waffle committed:
Important experimental changes:

-Removed the Chinese political struggle

-Changed adjacency rules of the Oresund strait to allow Britain to invade Sweden

Thinking-waffle committed:
netherlands will turn fascist more easily wallonia will get a much smaller war support boost if it's fighting for its life

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-1.7.1

Thinking-waffle committed:
I make stupid mistakes, that's why debugging exists

Thinking-waffle committed:
reality enables to debug even more

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-1.7.1

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-1.7.1

Thinking-waffle committed:
Chasseurs ardennais are now jaeger, Petrofina as a prototype for non synthetic oil companies

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-1.7.1
